contrib/mesos/docs/ha.md

```diff
@@ -1,4 +1,24 @@
-## High Availability
+# High Availability
+
+Kubernetes on Mesos supports 2 HA modes:
+
+* Hot-standby
+* Cold-standby
+
+In the hot-standby mode, all master components (apiserver, controller manager, and scheduler) actively run 
```

Comment:
I don't think we use leader election in k8sm's custom controller manager yet, so it doesn't support hot-standby at this point.